AGGREGATE.md

```diff
@@ -1,180 +1,359 @@
+*   A list of aggregation services which work together to aggregate the results. For initial experimentation we plan on allowing any origin to be an aggregation service, with the eventual goal of having an explicit allowlist of browser-trusted origins. Note that setting up a fallback mechanism in case these services fail is up to the services themselves. As long as the decryption key is available, reports for a particular service could be processed anywhere.
 
-Many advertisers use different notions of “viewability”, so it is important to support setting this attribute dynamically.
+Actions in the worklet won’t affect the embedder context, to avoid leaking information about the sensitive join of `triggerContext` and `attributionSourceContext`. This means that we need to be careful how errors are handled.
 
-Note that currently the event level API does not support view-through conversions, so viewed conversions will only send aggregate reports for now. (Note that we are actively exploring ideas to support view conversions in the event-level API too).
+### Aggregate attribution reports
 
-### Multi-touch attribution (MTA)
+Attribution reports will look very similar to [event-level reports](https://github.com/WICG/conversion-measurement-api#attribution-reports). They will be reported to the configured `attributionreportto` endpoint at the path `.well-known/attribution-reporting/report-aggregate-attribution`. The payload will be JSON encoded with the following scheme:
 
-Multi-touch attribution is the method in which multiple impressions leading up to a conversion are assigned credit. The event-level API currently only supports a “last-clicked” attribution model, where the last clicked impression received a credit of “100” and the impressions preceding it received a credit of “0”. However, supporting arbitrary rules-based models in a privacy preserving way is difficult in the event-level approach, since the credit associated with each conversion can leak more cross-site activity. These problems are mitigated with aggregation and differential privacy, so we can extend support for these other models here.
 
-This can be achieved by annotating the conversion registration URL with a new parameter:
-```
-agg-multi-touch-model=<linear,last-clicked,first-clicked,...>
+```json
+{
+  "source_site": "https://publisher.com",
+  "attribution_destination": "https://advertiser.com",
+  "scheduled_report_time": <timestamp in msec>,
+  "aggregation_service_payloads": [
+    {
+      "origin": "https://helper1.com",
+      "payload": <base64 encoded encrypted data>
```

Comment:
Should this have quotes? Presumably this is encoded as a string? 

ditto below

Reply:
done